albumentations/augmentations/transforms.py

```diff
@@ -332,32 +336,70 @@ class ImageCompression(ImageOnlyTransform):
     """
 
     class InitSchema(BaseTransformInitSchema):
-        quality_lower: int = Field(default=99, description="Lower bound on the image quality", ge=1, le=100)
-        quality_upper: int = Field(default=100, description="Upper bound on the image quality", ge=1, le=100)
+        _size_len = 2
+        quality_range: Optional[Union[Tuple[int, int], List[int]]] = Field(
```

Comment:
Let's use for quality range: `Optional[Tuple[int, int]]`

Historically we had an option in all transforms to specify a number or a Tuple and number is converted to Tuple before sampling.

But there is so many different ways to do this, say

`x => (-x, x)`
`x => (x, x)`
`x => (1 - x, 1+x)`

etc that it became a mess.

New policy that for all new interfaces, including refactoring of the old one like here, or creating new one - if it should be number => make it a number, it range => it should be tuple.

We have this in Countributor's guide, but I should probably extend it there:

```
Explicit Definitions: Parameters should be explicitly defined as either a single value or a tuple. This change avoids ambiguity and ensures that the intent and behavior of the transform are clear to all users.
```